From 160fdad2ae06e8fb5bb976b6da5fb0d6468604ac Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Tue, 20 Oct 2020 19:46:17 +0200
Subject: [PATCH 093/830] ANDROID: arm64: traps: export dump_backtrace symbol
X-NVConfidentiality: public

Export dump_backtrace symbol for modules to reference.

[added again as it moved around in 5.10-rc1]

Bug: 169899018
Cc: Jonglin Lee <jonglin@google.com>
Cc: Will Deacon <willdeacon@google.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I8448584ed8af204deaa991c5a60b04a39b47ac2f
---
 arch/arm64/kernel/stacktrace.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/kernel/stacktrace.c b/arch/arm64/kernel/stacktrace.c
index c445828..a336782 100644
--- a/arch/arm64/kernel/stacktrace.c
+++ b/arch/arm64/kernel/stacktrace.c
@@ -190,6 +190,7 @@ void dump_backtrace(struct pt_regs *regs, struct task_struct *tsk,
 
 	put_task_stack(tsk);
 }
+EXPORT_SYMBOL_GPL(dump_backtrace);
 
 void show_stack(struct task_struct *tsk, unsigned long *sp, const char *loglvl)
 {
-- 
2.7.4

